sky/global_user_state.py

```diff
@@ -516,7 +545,9 @@ def get_cluster_from_name(
         # we can add new fields to the database in the future without
         # breaking the previous code.
         (name, launched_at, handle, last_use, status, autostop, metadata,
-         to_down, owner, cluster_hash) = row[:10]
+         to_down, owner, cluster_hash, storage_mounts_metadata) = row[:11]
+        if not storage_mounts_metadata:
+            storage_mounts_metadata = pickle.dumps({})
```

Comment:
These two lines seem like will never happen

Reply:
This handles the case when user has rows from previous version that did not contain `storage_mounts_metadata` in the db. I'll leave a comment on this.